src/scripts/main.js

```diff
@@ -1,3 +1,292 @@
 'use strict';
 
-// write your code here
+let board;
+let score = 0;
+const ROWS = 4;
+const COLUMNS = 4;
+const startButton = document.getElementsByClassName('button start')[0];
+let gameEnded;
+
+startButton.addEventListener('click', setGame);
+
+function setGame() {
+  startGameMessage();
+  changeButtontoRestart(startButton);
+
+  board = [
+    [2, 0, 0, 2],
+    [0, 0, 0, 0],
+    [0, 0, 0, 0],
+    [0, 0, 0, 0],
+  ];
+  clearScore();
+  gameEnded = false;
+
+  let cellIndex = 0;
+
+  for (let i = 0; i < ROWS; i++) {
+    for (let j = 0; j < COLUMNS; j++) {
+      const num = board[i][j];
+      const cells = document.getElementsByClassName('field-cell')[cellIndex];
+
+      updateCell(num, cells);
+      cellIndex++;
+    }
+  }
+}
+
+function updateCell(num, cells) {
+  cells.innerText = '';
+  cells.classList.value = '';
+  cells.classList.add('field-cell');
+
+  if (num > 0) {
+    cells.innerText = num;
+
+    if (num <= 2048) {
+      cells.classList.add(`field-cell--${num}`);
+    }
+  }
+}
+
+document.addEventListener('keyup', function() {
+  winGameMessage(board);
+  loseGameMessage(board);
+});
```

Comment:
<img width="526" alt="image" src="https://github.com/mate-academy/js_2048_game/assets/70220667/e4e0afe1-eb84-41ae-bc90-819dc6cd5906">
<img width="484" alt="image" src="https://github.com/mate-academy/js_2048_game/assets/70220667/fa85044d-c68b-417c-8071-86780a04ece5">

we also can combine this two event listeners in one